File: morning snacks/Lcm.java
import java.util.Scanner;
    public class Gcd {
        public static void main(String...Odili) {
            Scanner input = new Scanner(System.in);

    System.out.println("Enter any number ");
        int firstNumber = input.nextInt();

    System.out.println("Enter another Number ");
        int secondNumber = input.nextInt();
            int gcd = 0; 
            int lcm = 0;        
   
        for(int count = 1; count <= firstNumber && count <= secondNumber; count ++){
            if(firstNumber % count == 0 && secondNumber % count == 0){  
                gcd = count;
                
                lcm = (int)firstNumber * secondNumber / gcd;
                

            }
                   
        }
            System.out.print("The Lcm is " + lcm);
        

        }
    }

File: morning snacks/lcm.py
numberOne = int(input("Enter any number "))

numberTwo = int(input("Enter another number "))

gcd = 0

lcm = 0

minimum = min(numberOne, numberTwo)

for number in range (1, minimum + 1):
    if(numberOne % number == 0 and numberTwo % number  == 0):
        gcd = number
        lcm = numberOne * numberTwo // gcd
print(lcm)
